### PR TITLE
Fix user service controller constructor

### PR DESCRIPTION
**Pull request by sbryzak**
This PR fixes the user service controller constructor.  

Related issue: https://github.com/redhat-developer/rh-che/issues/1409

### CODE REVIEW

File: controller/user_service_test.go [hunk 1]
```diff
@@ -16,6 +24,35 @@ var (
...
+	controller := NewUserinfoController(svc, s.Application, testtoken.TokenManager)
+	return svc, controller
+}
+
+func (s *UserServiceControllerTestSuite) TestShowOK() {
+	identity, err := testsupport.CreateTestIdentityAndUserWithDefaultProviderType(s.DB, "userTestShowUserOK"+uuid.NewV4().String())
+	require.Nil(s.T(), err)
+
+	svc, ctrl := s.SecuredController(identity)
+	test.ShowUserServiceInternalServerError(s.T(), svc.Context, svc, ctrl)
```
Comment (alexeykazakov) [inline controller/user_service_test.go]: It doesn't seem to test ShowOK :)

Comment (sbryzak) [reply controller/user_service_test.go]: Oops.. I've fixed the function name now.

Comment (alexeykazakov) [reply controller/user_service_test.go]: OK. But what about ShowOK test? We would have caught the bug if we had that but test in the first place, wouldn't we?

Comment (sbryzak) [reply controller/user_service_test.go]: We can't easily get an "OK" result from the controller.. we have extensive tests for the TenantService in `tenant_whitebox_test.go`, however it is testing an internally-created tenant service for which the `rest.HttpDoer` is replaced with a dummy object.  We don't have the mechanism to easily do this from the controller.

Comment (alexeykazakov) [reply controller/user_service_test.go]: You can use `factory.WithTenantService(serviceMock)`
See example https://github.com/fabric8-services/fabric8-auth/blob/master/controller/users_blackbox_test.go#L57

Comment (sbryzak) [reply controller/user_service_test.go]: I'm updating the test with this.

Comment (sbryzak) [reply controller/user_service_test.go]: All looks good now.

**Pull request merged**